server_config.cc

```diff
@@ -4,54 +4,92 @@
 #include <string>
 
 ServerConfig::ServerConfig(NginxConfig config) {
-  fillOutMap(config);
+  std::vector<std::string> basePath = {};
+  fillOutMap(config, basePath);
+  printPropertiesMap();
 }
 
-ServerConfig::~ServerConfig() {
-  // TODO
+ServerConfig::~ServerConfig() {}
```

Comment:
If the default (empty) destructor is sufficient, we could eliminate this line (and its declaration in the header) for now.